api/test/fakes/meeting.js

```diff
@@ -1,8 +1,10 @@
 const ObjectId = require('mongoose').Types.ObjectId;
 
 const meeting = ( opts ) => {
+  const randomName = ( Math.random() + 1 ).toString( 36 ).substring( 7 ); //rip faker
```

Comment:
Somebody else took over faker, its now at `@faker-js/faker`. Can you replace the faker in the root directory dev dependency with the new version?

Reply:
@thudsonbu done